### PR TITLE
[MI-113] Add incremental exports resource

### PR DESCRIPTION
**Pull request by miogalang**
Add resource for incremental export endpoints

/cc @jwswj @joseconsador @mmolina @atroche

### References
* JIRA: https://zendesk.atlassian.net/browse/MI-113

### Risks
* Low. You might not be able to call the incremental export endpoints

### CODE REVIEW

File: tests/Zendesk/API/UnitTests/IncrementalExportsTest.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        $this->getEndpointTest('users', 'incremental/users.json');
+    }
+
+    /**
+     * Test for the get endpoint using the given method and endpoint
+     *
+     * @param $method
+     * @param $endpoint
+     */
+    private function getEndpointTest($method, $endpoint)
```
Comment (joseconsador) [inline tests/Zendesk/API/UnitTests/IncrementalExportsTest.php]: This is awesome. Can we move this to BasicTest so it's accessible to all tests?

**Comment by miogalang** [comment]
@joseconsador :haircut:

**Comment by miogalang** [comment]
Reverting the commits pulling the function out. I want to use the utility function introduced in https://github.com/zendesk/zendesk_api_client_php/pull/151

**Comment by joseconsador** [comment]
:+1:

**Comment by jwswj** [comment]
I think this should should just be namespaced as `incremental` rather than `incrementalExport`

File: tests/Zendesk/API/UnitTests/IncrementalExportsTest.php (hunk 1 above)
Comment (jwswj) [inline tests/Zendesk/API/UnitTests/IncrementalExportsTest.php]: +1

**Comment by miogalang** [comment]
@jwswj I agree. We should follow the endpoint pattern instead of the documentation.

**Comment by miogalang** [comment]
:haircut: @jwswj

**Comment by jwswj** [comment]
:+1:

**Pull request merged**